docs/cog_guides/permissions.rst

```diff
@@ -0,0 +1,440 @@
+
+* ``<allow_or_deny>``: This should be one of "allow" or "deny".
+* ``<cog_or_command>``: The cog or command to add the rule to. This is case sensitive.
+* ``<who_or_what...>``: One or more users, channels or roles the rule is for.
+
+.. _permissions-command-permissions-canrun:
+
+""""""""""""""""""
+permissions canrun
+""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]permissions canrun <user> <command>
+
+**Description**
+
+Check if a user can run a command.
+
+This will take the current context into account, such as the
+server and text channel.
+
+**Arguments**
+
+* ``<user>``: The user to check permissions for.
+* ``<command>``: The command to check whether the user can run it or not.
+
+.. _permissions-command-permissions-clearglobalrules:
+
+""""""""""""""""""""""""""""
+permissions clearglobalrules
+""""""""""""""""""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]permissions clearglobalrules 
+
+**Description**
+
+Reset all global rules.
+
+.. _permissions-command-permissions-clearserverrules:
+
+""""""""""""""""""""""""""""
+permissions clearserverrules
+""""""""""""""""""""""""""""
+
+.. note:: |guildowner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]permissions clearserverrules 
+
+**Description**
+
+Reset all rules in this server.
+
+.. _permissions-command-permissions-explain:
+
+"""""""""""""""""""
+permissions explain
+"""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]permissions explain 
+
+**Description**
+
+Explain how permissions works.
+
+.. _permissions-command-permissions-removeglobalrule:
+
+""""""""""""""""""""""""""""
+permissions removeglobalrule
+""""""""""""""""""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]permissions removeglobalrule <cog_or_command> <who_or_what...>
+
+**Description**
+
+Remove a global rule from a command.
+
+**Arguments**
+
+* ``<cog_or_command>``: The cog or command to add the rule to. This is case sensitive.
+* ``<who_or_what...>``: One or more users, channels or roles the rule is for.
+
+.. _permissions-command-permissions-removeserverrule:
+
+""""""""""""""""""""""""""""
+permissions removeserverrule
+""""""""""""""""""""""""""""
+
+.. note:: |guildowner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]permissions removeserverrule <cog_or_command> <who_or_what...>
+
+**Description**
+
+Remove a server rule from a command.
+
+**Arguments**
+
+* ``<cog_or_command>``: The cog or command to add the rule to. This is case sensitive.
+* ``<who_or_what...>``: One or more users, channels or roles the rule is for.
+
+.. _permissions-command-permissions-setdefaultglobalrule:
+
+""""""""""""""""""""""""""""""""
+permissions setdefaultglobalrule
+""""""""""""""""""""""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]permissions setdefaultglobalrule <allow_or_deny> <cog_or_command>
+
+**Description**
+
+Set the default global rule for a command.
```

Comment:
This and the server equivalent should also mention that rules are for **cogs** or commands, but I can't think of a way to word it well since it shows up twice without swapping to "it" in the second reference.